A New Medbay for Boxstation

Maurukas opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
@Twaticus had created a reworked medbay for Box on a commit several months old but stalled on piping and utility issues. It's similar to the redesign Meta got a few months back. I've taken this layout and updated it with multilayer piping, power, restored sanity to disposals below medbay.

This contains the same number of surgical tables, stasis beds and operating computers as the current setup, just in a more spacious arrangement.

Medbay itself:
![BoxMedbay5](https://user-images.githubusercontent.com/66576896/84057742-f5958380-a96c-11ea-8e22-44b49b20cf5d.png)


## Why It's Good For The Game

Boxstation's current medbay is cramped, cluttered, has a large amount of wasted space, and is just hellish to work in. This will end the endless shoving in the 4x4 space around the current stasis beds.

## Changelog
:cl:
add: Boxstation and Icebox medbays have been completely reworked
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Maurukas committed:
Boxstation Medbay Redesign

Updates Boxstation's Medbay with a layout by Twat.

Maurukas committed:
Fix operating computers

Maurukas committed:
Add some additional roller beds

zxaber commented:
Should be added to Icebox too.

Maurukas commented:
I'm willing to give that a try. It looks like Icebox is a number of features behind Box though. AI chamber, grav gen, and multilayer piping all haven't been done on Icebox.

zxaber commented:
AI chamber can stay not moved plz.

The AI/gravgen change was something about making box "as it used to be". This is an entirely new feature.

Iatots commented:
That northern corridor looks like a nightmare of congestion.

GuillaumePrata commented:
I think it would be good to swap the right side doors of the Stasis room with glass windows, else people will use the stasis room as a hallway to move around medbay instead of taking 2 seconds longer to follow the proper hallway and the random shoving issue will stay the same.

Angustmeta commented:
Treatment center in particular looks way more comfortable than before. Some observations:

- Is that a glass door to the toilet in break room? Not sure if mapdiff shows that properly. I might read it wrong but psych office, break room and toilets should be opaque, for shady business. 

- Many rooms could use holopads. In particular pharmacy, CMO office, treatment center, medical storage. 

- Was the second syringe gun removed?

MrDoomBringer commented:
thoughts on something akin to this? Slightly widened north hallway and closed the door on the right
![image](https://user-images.githubusercontent.com/29008542/84072156-bb43da80-a99c-11ea-89a5-dfc1d770a6a9.png)
i used only the finest of mapping tools to achieve this thorough mockup

Maurukas committed:
Add holopads, additional syringe gun

Adds holopad to pharmacy
Adds holopad to medical storage
Adds holopad to cryo
Adds second syringe gun back to medical storage

Maurukas committed:
Expand medbay corridor

Maurukas commented:
@Angustmeta The doors are solid on the bathroom and psych office. Snuggle time with the moth plush and other such shady activities can remain confidential.

I re-added the second syringe gun, and added holopads to the pharmacy, CMO office, medbay storage and cryo. The treatment room has one, it's just hidden under the pipenetwork in the above .

I opted to widen the corridor by shaving a few tiles off the security post, rather than going into the exam room. Medbay may be ever so slightly less well defended as a result.
![BoxMedbay6](https://user-images.githubusercontent.com/66576896/84076652-02c16b00-a98b-11ea-927a-90e59ff1083f.png)

Maurukas committed:
Fix some rogue pipes

Maurukas committed:
Small fixes from test feedback

Minor area fixes
Meta-style defib mount setup - previously blocked by apcs/lights
fix cryo freezer orientation
Spooky secpost apc

Jared-Fogle commented:
The freezer isn't connected to the cryo pipes. You need to rotate it with a screwdriver and a wrench.

Maurukas committed:
Access helpers on medbay doors

Get out of medbay. No. Get out. Now.

Maurukas committed:
Add new medbay to icebox

Maurukas commented:
Multiple fixes, including doors and cryo. As requested by @zxaber, I've added this to Icebox as well and included it in this PR.

Maurukas committed:
Fix conflicts with #51457

Added pipes/disposals/wires and a door

Maurukas commented:
After clearing those conflicts, I believe I am finished with this and it is ready to merge.

Maurukas committed:
Oops. Space turfs.

Rohesie commented:
Looks nice, but more conflicts.

Maurukas committed:
Deal with #51642 - please work

TheVekter commented:
Oh thank fuck you moved the chem factory somewhere not stupid
